src/Components/UI/Calculator.css

```diff
@@ -0,0 +1,53 @@
+.calculator {
+  width: 100%;
+  background-color: rgb(148, 145, 145);
+}
+
+.calcInput {
+  width: 100%;
+  height: 150px;
+  background-color: rgb(161, 161, 161);
+  color: white;
+  font-size: 60px;
+  text-align: right;
+  box-sizing: border-box;
+  padding-right: 70px;
+  border: none;
+}
+
+.make-flex {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+}
+
+.row {
+  display: flex;
+}
+
+.calcButton {
+  color: black;
+  background-color: rgb(248, 248, 248);
+  border: 1px solid rgb(222, 222, 222);
+  font-size: 40px;
+  width: 400px;
```

Comment:
- [ ] Kindly make the `width: 25%;` for the `calcButton` so the design looks good and the external space on the right is gone.

![Capture](https://user-images.githubusercontent.com/33752735/197840035-5dee7ea5-82be-4632-be32-05d273a4cfc2.PNG)